store/sqlstore/channel_store_categories.go

```diff
@@ -436,7 +447,7 @@ func (s SqlChannelStore) GetSidebarCategory(categoryId string) (*model.SidebarCa
 	}
 
 	if _, err = s.GetReplica().Select(&categories, sql, args...); err != nil {
-		return nil, store.NewErrNotFound("SidebarCategories", categoryId)
+		return nil, errors.Wrap(err, "hello error")
```

Comment:
What is "hello error"? 😅

Reply:
It is called a "left-during-debugging" error :rofl: Fixing.